lib/web/apiserver.go

```diff
@@ -339,6 +343,8 @@ func (c *Config) SetDefaults() {
 	if c.PresenceChecker == nil {
 		c.PresenceChecker = client.RunPresenceTask
 	}
+
+	c.FeatureWatchInterval = cmp.Or(c.FeatureWatchInterval, DefaultFeatureWatchInterval)
```

Comment:
FYI, `cmp.Or` requires Go 1.22 or later.

We build with Go 1.22, so that's why the build hasn't failed.

We do however set the language level to Go 1.21 in branch/v15, which generates a warning.

```
module github.com/gravitational/teleport

go 1.21

toolchain go1.22.10
```

At this point, we might as well just update the go directive to 1.22 here.